app/frontend/src/features/communities/CommunityPage/PlacesSection.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import Alert from "../../../components/Alert";
+import CircularProgress from "../../../components/CircularProgress";
+import HorizontalScroller from "../../../components/HorizontalScroller";
+import IconButton from "../../../components/IconButton";
+import { InfoIcon, MoreIcon } from "../../../components/Icons";
+import TextBody from "../../../components/TextBody";
+import { Community } from "../../../pb/communities_pb";
+import { routeToCommunityPlaces } from "../../../routes";
+import { useListPlaces } from "../useCommunity";
+import { useCommunityPageStyles } from "./CommunityPage";
+import PlaceCard from "./PlaceCard";
+import SectionTitle from "./SectionTitle";
+
+export default function PlacesSection({
+  community,
+}: {
+  community: Community.AsObject;
+}) {
+  const classes = useCommunityPageStyles();
+
+  const {
+    isLoading: isPlacesLoading,
+    error: placesError,
+    data: places,
+    hasNextPage: placesHasNextPage,
+  } = useListPlaces(community.communityId);
+
+  return (
+    <>
+      <SectionTitle icon={<InfoIcon />}>Places</SectionTitle>
+      {placesError && <Alert severity="error">{placesError.message}</Alert>}
+      {isPlacesLoading && <CircularProgress />}
+      <HorizontalScroller className={classes.cardContainer}>
+        {
+          //Is there a better way to check for empty state?
+          places &&
```

Comment:
Probably not, considering the `pages[0].<typeList>` changes depending on the data type. Could abstract this into a helper but might be tricky to get it type safe without delving into some hairy generics 😅